SerialPrograms/Source/CommonFramework/AudioPipeline/AudioNormalization.h

```diff
@@ -63,17 +65,17 @@ PA_FORCE_INLINE int16_t byte_swap(int32_t x){
 
 template <typename Type>
 void normalize_audio_le(float* out, const Type* in, size_t count){
-    const float rcp = 1. / std::numeric_limits<Type>::max();
-    const float sub = std::is_unsigned<Type>::value ? 1 : 0;
+    const float rcp = (std::is_unsigned<Type>::value ? 2.0f : 1.0f) / std::numeric_limits<Type>::max();
```

Comment:
you should add
`#include <type_traits>`
in this file ^^

Reply:
Also needs `<stdint.h>`.
```
#include <stdint.h>
#include <limits>
#include <type_traits>
#include "Common/Compiler.h"
```